src/main/java/moze_intel/projecte/config/ClientConfig.java

```diff
@@ -40,6 +42,9 @@ public class ClientConfig extends BasePEConfig {
 		pulsatingOverlay = CachedBooleanValue.wrap(this, builder
 				.comment("The Philosopher's Stone overlay softly pulsates")
 				.define("pulsatingOverlay", false));
+		condenserEmcDisplayMode = CachedIntValue.wrap(this, builder
+				.comment("Sets how EMC is displayed inside Energy Condensers.\n0: Shows how much EMC is stored or how much is required to make the item if there is Excess stored EMC. (EE2 Style)\n1: Will show the stored amount of EMC and render green if the value exceeds required amount.\n2: If the stored EMC value is greater then required it will display the total EMC stored seperately in green.\nNote: If the value displayed would overrun the edge of the GUI it will instead state 'Overflow'.")
+				.define("showTotalEMCInCondenser", 0));
```

Comment:
I feel like this would make more sense just using an enum for the config. General handling would be making a new config value type that basically is a copy of https://github.com/mekanism/Mekanism/blob/1.18.x/src/main/java/mekanism/common/config/value/CachedEnumValue.java. I also believe you can pass multiple strings to `comment` to properly apply them on new lines rather than injecting manual `\n` declarations

Reply:
Fair Enough. Ill adjust that too along with the Translations.